modules/investigating-kubelet-api-installation-issues.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// *support/troubleshooting-installations.adoc
+
+[id="investigating-kubelet-api-installation-issues_{context}"]
+= Investigating master node kubelet and API server issues
+
+To investigate master node kubelet and API server issues during installation, check DNS, DHCP and load balancer functionality. Also, verify that certificates have not expired.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* The `ssh-agent` process is active on your computer, and you provided both the `ssh-agent` process and the installation program the same SSH key.
+* If you tried to install a cluster on infrastructure that you provisioned, you must have the fully-qualified domain names of the control plane, or master, machines.
+* You have system administrator privileges.
+
+.Procedure
+
+. Verify the API server's DNS record directs kubelet on master nodes to https://api-int.<cluster_name>.<base_domain>:6443. Ensure that the record references the load balancer.
```

Comment:
This URL should probably not be formatted as a URL, since clicking it will not take you to an appropriate location.  You should probably put escapte it and consider putting it in backticks.

So either:
```
\https://api-int.<cluster_name>.<base_domain>:6443
```
or in backticks:

```
[x-]`https://api-int.<cluster_name>.<base_domain>:6443`
```

Reply:
Thank you, I will keep that in mind in the future. Resolved.